tests/test_exceedance.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import os
+import pandas as pd
+import pytest
+import xarray as xr
+
+from climakitae import threshold_tools
+
+#------------- Data for testing -----------------------------------------------
+
+@pytest.fixture
+def T2_monthly(test_data):
+    """ Monthly RAINC data for one scenario and one simulation 
+    (pulled from the general test data set)"""
+    return test_data["RAINC"].isel(scenario=0, simulation=0)
```

Comment:
It's probably a good time to add the test dataset to the repo now, since we are close to merging and want to ensure the tests pass. That is, unless you are still making changes to that dataset?

Reply:
adding now!